releasenotes/notes/update-cs-csdg-in-equivalence-library-c1e70d3246f4aa6d.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
```

Comment:
Isn't this more of an `upgrade` or `features`? It wasn't wrong before, it's just better now 🙂